public/js/problem_add.js

```diff
@@ -27,36 +27,44 @@ $('.submit').on('click', function () {
 	// creating question object
 	const ques = {};
 
-	ques.Name = $("#QuesName").val();
-	ques.Description = $("#Description").val();
-	ques.InputFormat = $("#InputFormat").val();
-	ques.OutputFormat = $("#OutputFormat").val();
-	ques.Constraints = $("#Constraints").val();
-	ques.SampleInput = $("#SampleInput").val();
-	ques.SampleOutput = $("#SampleOutput").val();
-	ques.Explanation = $("#Explanation").val();
-	ques.Difficulty = $("#DifficultyLevel").val();
-	ques.Setter = $("#ProblemSetter").val();
-	ques.Time = $("#TimeLimit").val();
-	ques.Memory = $("#MemoryLimit").val();
-	ques.Tags = $("#Tags").val();
-	ques.Editorial = $("#Editorial").val();
+	ques.name = $("#QuesName").val();
+	ques.description = $("#Description").val();
+	ques.inputFormat = $("#InputFormat").val();
+	ques.outputFormat = $("#OutputFormat").val();
+	ques.constraints = $("#Constraints").val();
+	ques.sampleInput = $("#SampleInput").val();
+	ques.sampleOutput = $("#SampleOutput").val();
+	ques.explanation = $("#Explanation").val();
+	ques.difficulty = $("#DifficultyLevel").val();
+	ques.problemSetter = $("#ProblemSetter").val();
+	ques.timeLimit = $("#TimeLimit").val();
+	ques.memoryLimit = $("#MemoryLimit").val();
+	ques.tags = $("#Tags").val().split(",");
```

Comment:
Add `.trim()` also at the end in line 42 to remove the leading and trailing spaces.